### DISCUSSION TITLE
Any branding logo other than ".svg" breaks Nova with no error message

### DISCUSSION THREAD

**Discussion by joncarlson** [laravel/nova-issues#5967] [Ideas & Feature Requests]
- Laravel Version: 10.28.0
- Nova Version: 4.27.13
- PHP Version: 8.2.9
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:

Trying to customize branding. Didn't notice that SVG is a required logo type, I was trying to use a PNG. The requirement is ok, but a completely broken page without an error message is a bug.

### Detailed steps to reproduce the issue on a fresh Nova installation:

- upload a PNG or JPG to /public/
- in config/nova.php, change the "branding"->"logo" property to `public_path('/filename.png')`
- visit Nova, it won't load at all

**Comment by holubar** [comment]
I would really appreciate if there was an easier way to use custom logo. I tried multiple online tools to convert my png to svg, none of the svgs work in nova. I wish this was easier.
